linux_os/guide/services/sssd/sssd_enable_smartcards/oval/shared.xml

```diff
@@ -27,7 +27,7 @@
   </ind:textfilecontent54_test>
   <ind:textfilecontent54_object id="obj_sssd_enable_smartcards" version="1">
     <ind:filepath>/etc/sssd/sssd.conf</ind:filepath>
-    <ind:pattern operation="pattern match">^[\s]*\[pam](?:[^\n\[]*\n+)+?[\s]*pam_cert_auth[\s]*=[\s]*(?i)true\s*$</ind:pattern>
+    <ind:pattern operation="pattern match">^[\s]*\[pam](?:[^\n\[]*\n+)+?[\s]*pam_cert_auth[\s]*=[\s]*(?i)[Tt]rue\s*$</ind:pattern>
```

Comment:
The `(?i)` in the regular expression turns on case insensivity for the remainder of the pattern, therefore, you don't need to use `[Tt]` in the pattern, both lowercase and uppercase variant will be matched anyway.